network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -578,6 +578,24 @@ public void hvdcLineTest() {
         }
     }
 
+    @Test
+    public void testHvdcLineRemove() {
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+            Network network = NetworkStorageTestCaseFactory.create(service.getNetworkFactory());
+            service.flush(network);
+        }
+
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+            Map<UUID, String> networkIds = service.getNetworkIds();
+            assertEquals(1, networkIds.size());
+            Network readNetwork = service.getNetwork(networkIds.keySet().stream().findFirst().get());
+            assertEquals(1, readNetwork.getHvdcLineCount());
+            readNetwork.getHvdcLine("HVDC1").remove();
+            assertEquals(0, readNetwork.getHvdcLineCount());
+            service.flush(readNetwork);
+        }
+    }
```

Comment:
It might be useful to read the network from database again and ensure that the flush of the network (the one made just after hvdcLine removal) correctly persisted the hvdcLine removal.
Maybe also add a test where an hvdcLine is removed and another one with the same name is created before applying the flush (see the testDanglingLineRemove).

Reply:
Done